tamr_unify_client/models/taxonomy/category.py

```diff
@@ -0,0 +1,49 @@
+from tamr_unify_client.models.base_resource import BaseResource
+
+
+class Category(BaseResource):
```

Comment:
It looks like `Category` wants to be its own resource under `models`.
so `tamr_unify_client/models/category/resource.py` and `tamr_unify_client/models/collection.py`